components/Reference/columns.tsx

```diff
@@ -2,9 +2,13 @@ import { Row } from 'react-table'
 
 import { StackBox } from 'components/ui'
 
-const filter = (rows: Row[], id: string, filterValue: string) => {
-  const re = new RegExp(`${filterValue}`, 'i')
-  return rows.filter((row: any) => row.original[id].match(re))
+// Possible fields are defined in `Opcodes.json`
+type OpcodeRow = Row<Record<string, string | undefined>>
```

Comment:
Not able to get this fully typed as it depends on the json file, but this is better than the `any` type